_sass/_redesign-platform.scss

```diff
@@ -156,138 +233,748 @@ $platform-page-content-reduce-padding-width: 1199px;
+                            flex-basis: unset;
+                            flex-grow: unset;
+                            flex-shrink: unset;
+                            widtH: 100%;
+                        }
+                        width: 100%;
+                        > .redesign-button--wrapper {
+                            width: 100%;
+                            box-sizing: border-box;
+                            &.redesign-button--wrapper__complex-content {
+                                > .redesign-button--anchor {
+                                    width: 100%;
+                                }
+                            }
+                        }
+                        > .redesign-button-pair--wrapper {
+                            > .redesign-button--wrapper {
+                                &.redesign-button--wrapper__text-only {
+    
+                                }
+                            } 
+                        }
+                    }
+                }
             }
         }
-        > hgroup {
-            flex-basis: 1265px;
-            height: 400px;
+    }
+}
+
+.platform-landing-page {
+    .landing-page-content__right-panel {
+        > .landing-page-content__right-panel--content {
+            > .landing-page-content__right-panel--content--body {
+                min-height: 485px;
+            }
+            > .landing-page-content__right-panel--content--side-panel {
+                min-height: 485px;
+            }
         }
     }
-    .platform-landing-page--details {
+}
+
+.platform-category-page {
+    @include thick-edge-top;
+    .platform-category-page--overview {
+        @include page-element-padding;
+        > .platform-category-page--hero {
+            height: auto;
+            aspect-ratio: 2 / 1;
+            @media screen and (min-width: 835px) {
+                max-width: 1200px;
+            }
+            @media screen and (max-width: 834px) {
+                max-width: 100%;
+            }
+            position: relative;
+            box-sizing: border-box;
+            margin: 0;
+            padding: 0;
+            > .landing-page-hero--images {
+                position: absolute;
+                top: 0;
+                left: 0;
+                right: 0;
+                bottom: 0;
+                margin: 0;
+                padding: 0;
+                width: 100%;
+                height: 100%;
+                > .landing-page-hero--images--image {
+                    position: absolute;
+                    top: 0;
+                    left: 0;
+                    right: 0;
+                    bottom: 0;
+                    margin: 0;
+                    padding: 0;
+                    width: 100%;
+                    height: 100%;
+                    &.landing-page-hero--images--image__desktop {
+                        > img {
+                            object-fit: fill;
```

Comment:
There are 10 uses of `object-fit: fill` in 
* `_sass/_redesign-platform.scss`
* `_sass/_redesign-platform.scss`
* `_sass/_redesign-platform.scss`

which will result in distorted and unpleasant rendering. `object-fit: cover` will be better.

That said, I am not a fan of the `position: absolute` and the fixed height and width (dictated by a fixed aspect ratio).

Reply:
I agree. We need a specification for how images behave responsively. Whether to maintain aspect ratio with rules about how to position, and what to fill in the space, or to crop. I have reached out for design guidance on this. The really high touch graphics look great at or near their source geometry, but across variable viewports things get dodgy.